### ISSUE TITLE
Include the Windows.UI.Xaml.Shapes API

### ISSUE THREAD

**Issue by ChrisCoble** [Microsoft/xaml-standard#36]
Please consider including some additional things besides the basic forms support for the initial release.  This would allow far more interesting experiences.

Including the Windows.UI.Xaml.Shape API would of course need the support for brushes, pen's, etc.

**Comment by BillHenning** [comment]
Yes this is a must-have.

**Comment by michael-hawker** [comment]
Yeah, it'd tie into the Windows.UI.Xaml.Media API as well to grab Transforms, Lines/Paths, and Brushes.

Like Composite/Matrix/Rotate/Scale/Skew/Translate Transform, SolidColorBrush, ImageBrush, and LinearGradiantBrush.  

It'd be really nice to see **RadialGradientBrush** make a comeback as a ton of Silverlight and WPF graphics based projects used that brush for their vector graphics.  It'd at least help make porting older projects to the standard easier.
